src/Dialect/ONNX/DialectBuilder.cpp

```diff
@@ -146,18 +146,19 @@ Value OnnxBuilder::mul(Type resultType, Value A, Value B) const {
       resultType, toTensor(A), toTensor(B));
 }
 
-Value OnnxBuilder::pad(Type outputType, Value input, Value pads,
-    Value constantValue, std::string mode) const {
+Value OnnxBuilder::pad(
+    Value input, Value pads, Value constantValue, std::string mode) const {
+  Type elementType = input.getType().cast<ShapedType>().getElementType();
```

Comment:
You can use `getElementType(input.getType())` from `src/Support/TypeUtilities.hpp`.

Reply:
Done. Thanks!